drivers/wireless/ieee80211/bcm43xxx/bcmf_chip_43362.c

```diff
@@ -45,39 +45,41 @@ extern const uint8_t bcm43362_firmware_image[];
 extern const unsigned int bcm43362_firmware_image_len;
 #endif
 
-const struct bcmf_sdio_chip bcmf_43362_config_sdio =
+struct bcmf_sdio_chip bcmf_43362_config_sdio;
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+void bcmf_43362_init(struct bcmf_sdio_chip *conf)
 {
+  uint32_t *core = conf->core_base;
+
   /* General chip stats */
 
-  .ram_base = 0,
-  .ram_size = 0x3c000,
+  conf->ram_base = 0;
```

Comment:
can we move ram_base/ram_size to the first instead?

Reply:
They are actually the first. What do you mean?

Reply:
If they are first, we can directly initialize the struct:
```
{
  0,
  0x3c000,
}
```
to conform c89 spec.

Reply:
I have updated the patch (disregarding C89 so far).